lib/three_scale/conversions.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module ThreeScale
+  module Conversions
```

Comment:
I would prefer to namespace it within Money, as it's about money, and not just any conversion.

But I guess that would clash with the class ThreeScale::Money ... so, well, not strongly objecting this naming.

Reply:
Yeah, that's an option too (and should be quite easy).
Up to you, I don't mind either way (keeping it like this, or renaming)

Reply:
I'll modify this in https://github.com/3scale/porta/pull/3339, since this is a minimal change, so I don't dismiss this approval.